README.md

```diff
@@ -21,11 +21,16 @@ It currently verifies:
 
 
 ## Makefile Targets
-- `make build`: Builds executable
-- `make test`: `go test $(GOFLAGS)`
-- `make build-push`: Builds and pushes image from build/ to ` quay.io/app-sre/osd-network-verifier:$(IMAGE_URI_VERSION)`
-- `make skopeo-push`: (TODO add)
-- 
+ONV uses openshift/boilerplate https://github.com/openshift/boilerplate
+
+Some useful targets
+- `make osd-container-image-build` : Build the default container at build/Dockerfile and tag it based on the commit. Specify DOCKERFILE and IMAGE_URI to build other containers.
+- `make osd-container-image-push` : Push the default container. 
+  - To run locally, `make REGISTRY_USER=<quay-username> REGISTRY_TOKEN=<quay-token>  osd-container-image-build-push`
+- `make prow-config` : Updates the corresponding Prow config file in openshift/release to run make test on merge requests. This test make target should be defined by the consumer. If this is a new repository it should be onboarded to openshift/release first before this is run.
```

Comment:
I am not sure we should document this command as it is only an onboarding helper generating a part of the global prow config (specific to 1 convention). 
If we want to keep it documented, we should be clearer on what it does and what it doesn't do.